src/share/classes/com/sun/javatest/regtest/tool/Tool.java

```diff
@@ -2336,12 +2339,20 @@ public enum ReportMode { NONE, EXECUTED, FILES, ALL_EXECUTED, ALL }
     private static final String MANUAL    = "manual";
 
     private static final String[] DEFAULT_UNIX_ENV_VARS = {
-        "DISPLAY", "GNOME_DESKTOP_SESSION_ID", "HOME", "LANG",
-        "LC_ALL", "LC_CTYPE", "LPDEST", "PRINTER", "TZ", "XMODIFIERS"
+            "DBUS_SESSION_BUS_ADDRESS", "DESKTOP_SESSION", "DISPLAY",
+            "GDM_SESSION", "GNOME_DESKTOP_SESSION_ID", "GNOME_SHELL_SESSION_MODE",
```

Comment:
Name should be `GDMSESSION`, not `GDM_SESSION`

Reply:
Will fix and push